.analysis_options

```diff
@@ -1,4 +1,6 @@
 analyzer:
+  language:
+    enableAssertInitializer: true
```

Comment:
Hmmmm.  Can we move this to an integration test?  Look in `test/_data` at the various projects.  Maybe we can define analysis options for a single project there?  Alternatively, we could analyze a single directory using `--assert-initializer` from `integration_test.dart`?